charts/temporal/values.yaml

```diff
@@ -263,6 +263,7 @@ server:
     tolerations: []
     affinity: {}
     additionalEnv: []
+    additionalEnvSecretName: ""
```

Comment:
What do you think about `additionalEnvFromSecret`?

Reply:
The value would be the name.

Reply:
`additionalEnvSecretName` is more clear it needs secret name here.